tsconfig.base.json

```diff
@@ -16,6 +16,9 @@
     "baseUrl": ".",
     "paths": {
       "@api/*": ["apps/cugetreg-api/src/*"],
+      "@cugetreg/codegen": ["libs/codegen/src"],
+      "@cugetreg/codegen/future": ["libs/codegen/src/future"],
```

Comment:
do we need to create a separate path just for future tho?

Reply:
We should separate because `future` is temporary.

Reply:
but it behaves the exact same way, no? What benefits does separating them gives? When we remove future, we'll have to remove `@cugetreg/codegen/future` anyways